src/MSBuild.Sdk.SqlProj/Sdk/Sdk.props

```diff
@@ -39,6 +39,7 @@
   <ItemGroup>
     <Content Include="**/*.sql">
       <Pack>False</Pack>
+      <SuppressTSqlWarnings></SuppressTSqlWarnings>
```

Comment:
Yes, it is nessesary to add `SuppressTSqlWarnings` metadata for all default `Content` items. Because if we doesn't add it, and make reference to it, MSBuild fail to process